src/vc_sqnm/mrchem_optimizer.hpp

```diff
@@ -0,0 +1,136 @@
+#include <iostream>
+#include <string>
+
+#include <MRCPP/Timer>
+#include <MRCPP/Parallel>
+
+#include "driver.h"
+#include "mrchem.h"
+#include "mrenv.h"
+#include "version.h"
+
+#include "chemistry/Molecule.h"
+// #include "chemistry/PhysicalConstants.h"
+#include "vc_sqnm/periodic_optimizer.hpp"
+
+#include <Eigen/Dense>
+
+using json = nlohmann::json;
+using namespace mrchem;
+
+Eigen::MatrixXd getPositions(json mol_inp) {
+    Molecule mol;
+    driver::init_molecule(mol_inp, mol);
+    Eigen::MatrixXd pos(3, mol.getNNuclei());
+
+    int i = 0;
+    for (const auto &coord : mol_inp["coords"]) {
+        double xyz[3];
+        xyz[0] = coord["xyz"][0];
+        xyz[1] = coord["xyz"][1];
+        xyz[2] = coord["xyz"][2];
+
+        pos.col(i) = Eigen::Vector3d(xyz);
```

Comment:
I think this does it:
```suggestion
        pos.col(i) << coord["xyz"][0], coord["xyz"][1], coord["xyz"][2];
```